Documentation/operations/upgrade.rst

```diff
@@ -366,6 +367,12 @@ The following metrics have been removed:
 * Label ``subnetId`` and ``availabilityZone`` in ``cilium_operator_ipam_available_ips_per_subnet`` are removed. Please
   use label ``subnet_id`` and ``availability_zone`` instead.
 
+New Metrics
+~~~~~~~~~~~
+
+  * ``cilium_datapath_conntrack_dump_resets_total`` Number of conntrack dump resets. Happens when a BPF entry gets removed
+    while dumping the map is in progress.
+
```

Comment:
I wonder, does it count as a new metrics if we end up renaming `cilium_datapath_errors_total`? :thinking: I'm fine either way.

Reply:
@qmonnet as I also added in Removed Metrics/Label the `datapath_errors` and what to replace it with, I told myself it's cleaner this way: having it in both sections.

Reply:
Ok fair enough